test/run.sh

```diff
@@ -25,10 +26,11 @@ Options:
   -v DIR   The venv directory. (${RF_VENV})
   -i PATH  The variables file. (${RF_VARIABLES})
   -k SKIP_TESTS      Comma separated list of tests to skip.
+  -r RANDOMIZE       Define RF Test order (${RANDOMIZE})
```

Comment:
The flag description should be more specific on what it does, and what it accepts. From the `robot --help` description of the randomize flag:

```
--randomize all|suites|tests|none  Randomizes the test execution order.
                      all:    randomizes both suites and tests
                      suites: randomizes suites
                      tests:  randomizes tests
                      none:   no randomization (default)
                      Use syntax `VALUE:SEED` to give a custom random seed.
                      The seed must be an integer.
                      Examples: --randomize all
                                --randomize tests:1234

```